plugins/kubernetes/README.md

```diff
@@ -136,3 +136,14 @@ Kubernetes::ReleaseGroup
         |
         -> Kubernetes Pods (how ever many replicas specified)
 ```
+### CLair security scans
+
+To use hyperclair tool (command-line tool for Clair and Docker Registry) for security scans we should supply environment variables:
+
+```
+CLAIR_SCAN_ENABLED = [ true | false ]
```

Comment:
how about when the path is set, then it's enabled ?

this could be good enough:

```
To use hyperclair tool (command-line tool for Clair and Docker Registry) for security scans, configure environment variables 
`CLAIR_SCRIPT = /filesystem/path/to/hyperclair`
```